### ISSUE TITLE
contribute-list

### ISSUE THREAD

**Issue by anushkrishnav** [anushkrishnav/PyAutomate#13]
Contribute List
reference : 
https://github.com/marketplace/actions/contribute-list

**Comment by UdokaVrede** [comment]
Hi can I take this up

**Comment by anushkrishnav** [comment]
Hey the issue is solved. Can you work on All contributors?

**Comment by UdokaVrede** [comment]
what about this issue please
Any link please

**Comment by anushkrishnav** [comment]
> what about this issue please
> Any link please

Yehh check this https://github.com/marketplace/actions/contribute-list .
Output : https://github.com/akhilmhdh/contributors-readme-action/blob/master/contributors.md

**Comment by tdserapio** [comment]
May I contribute on something?

**Comment by anushkrishnav** [comment]
> May I contribute on something?

Yes open an issue on something which you can automate using python to make life easier . Sending bulk mail to your friends or stock market price tracker (this would be fun )
